executor/explain_test.go

```diff
@@ -174,10 +174,10 @@ func (s *testSuite) TestExplain(c *C) {
 		{
 			"select * from t1 left join t2 on t1.c2 = t2.c1 where t1.c1 > 1",
 			[]string{
-				"TableScan_8", "TableScan_10", "HashLeftJoin_7",
+				"TableScan_7", "TableScan_10", "HashLeftJoin_9",
```

Comment:
Because the position of calling convert2IndexNestedLoopJoinLeft/Right is changed.